packages/mdg:geolocation/README.md

```diff
@@ -10,10 +10,22 @@ There are currently no options to set. Every method is reactive using [Tracker](
 
 Returns the [position error](https://developer.mozilla.org/en-US/docs/Web/API/PositionError) that is currently preventing position updates.
 
-### Geolocation.currentLocation()
+### Geolocation.currentLocation(options)
 
 Returns the [position](https://developer.mozilla.org/en-US/docs/Web/API/Position) that is reported by the device, or null if no position is available.
 
-### Geolocation.latLng()
+The `options` parameter is optional; if provided it as if `Geolocation.setOptions` was called before returning the position.
 
-A simple shorthand for currentLocation() when all you need is the latitude and longitude. Returns an object that has `lat` and `lng` keys.
\ No newline at end of file
+### Geolocation.latLng(options)
+
+A simple shorthand for currentLocation() when all you need is the latitude and longitude. Returns an object that has `lat` and `lng` keys.
+
+The `options` parameter is optional; if provided it as if `Geolocation.setOptions` was called before returning the position.
+
+### Geolocation.setOptions(options)
+
+Provide [PositionOptions](https://developer.mozilla.org/en-US/docs/Web/API/PositionOptions) to manage power consumption on mobile devices.  The options can be reactive.
```

Comment:
See the simple-map example above.  You can call:
```
var loc = Geolocation.latLng({ enableHighAccuracy: !!Session.get("highAccuracy") });
```
and the reactive-dependency magic will work to ensure that `loc` is properly updated if the `Session` variable changes (or if the location changes, naturally).